fastly/block_fastly_service_v1_dynamicsnippet.go

```diff
@@ -97,7 +78,7 @@ func processDynamicSnippet(d *schema.ResourceData, conn *gofastly.Client, latest
 	return nil
 }
 
-func readDynamicSnippet(conn *gofastly.Client, d *schema.ResourceData, s *gofastly.ServiceDetail) error {
+func (h *DynamicSnippetServiceAttributeHandler) Read(d *schema.ResourceData, s *gofastly.ServiceDetail, conn *gofastly.Client) error {
```

Comment:
I also want to point out how much I appreciate the constant use of `h` as the method receiver param name. This will make templating future additions easier. Thank you!